maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -218,6 +243,53 @@ private boolean connectMySqlDatabase(Server server) throws MojoExecutionExceptio
         }
     }
 
+    private void importMysqlDb(Server server, String sqlScriptPath) {
+        wizard.showMessage("Importing a database from " + sqlScriptPath + "...");
+        String uri = server.getDbUri().replace("@DBNAME@", server.getDbName());
+        Connection connection = null;
+        try {
+            connection = DriverManager.getConnection(uri, server.getDbUser(), server.getDbPassword());
+            Reader scriptReader = getSqlScriptReader(sqlScriptPath);
+            ScriptRunner scriptRunner = new ScriptRunner(connection);
+            //we don't want to display ~5000 lines of queries to user if there is no error
+            scriptRunner.setLogWriter(new PrintWriter(new NullOutputStream()));
+            scriptRunner.setStopOnError(true);
+            scriptRunner.runScript(scriptReader);
+            scriptRunner.closeConnection();
+            wizard.showMessage("Database imported successfully.");
+        } catch (SQLException e) {
+            try {
+                if(connection!=null){
+                    connection.close();
+                }
+            } catch (SQLException e1) {
+                getLog().error(e.getMessage());
+            }
+        }
+    }
+
+    private Reader getSqlScriptReader(String sqlScriptPath){
+        InputStream stream;
+        if(sqlScriptPath.startsWith(CLASSPATH_SCRIPT_PREFIX)){
+            String resource = sqlScriptPath.replace(CLASSPATH_SCRIPT_PREFIX, "");
+            stream = (Setup.class.getClassLoader().getResourceAsStream(resource));
+        } else {
+            File scriptFile = null;
+            File relative = new File(System.getProperty("user.dir"), sqlScriptPath);
```

Comment:
I think you can just use new File(sqlScriptPath) for both aboslute and relative. The class will determine if it is relative or absolute based on checking if the path starts from '/' or 'c:/'